examples/jenkins/pipeline/jenkinstemplate.json

```diff
@@ -37,6 +37,10 @@
       "apiVersion": "v1",
       "metadata": {
         "name": "${JENKINS_SERVICE_NAME}",
+        "annotations": {
+          "service.alpha.openshift.io/dependencies": "[{\"name\": \"jenkins-jnlp\", \"namespace\": \"\", \"kind\": \"Service\"}]",
```

Comment:
we care about the resource type so we know where to go look for jenkins-jnlp object.